.cascade-code/Chapman.edu/_cascade/formats/Homepage_Blog_Stories

```diff
@@ -180,8 +183,15 @@
 
       <p>$topStoryExcerpt</p>
 
-      #if ( $topStoryLinkText != "" )
-        <p><a href="$topStoryLink">$topStoryLinkText »</a></p>
+      #if ( $topStoryExtraLinkText != "" )
+        #if ( $topStoryExtraExternalLink != '' )
+            #set ($topStoryExtraLink = $topStoryExtraExternalLink )
+        #elseif ( $topStoryExtraInternalLinkPath != '' && $topStoryExtraInternalLinkPath != '/' )
+            #set ($topStoryExtraLink = $topStoryExtraInternalLinkPath )
+        #else
+            #set ($topStoryExtraLink = '#')
+        #end
+        <p><a href="$topStoryExtraLink">$topStoryExtraLinkText »</a></p>
```

Comment:
Do you want to use `${_EscapeTool.xml($topStoryExtraLinkText)` to allow special characters? I was able to break it by adding a & in the link text. Up to you, we could just tell the web coordinators not to use any special characters.

Reply:
yes I will add that. The link text wasn't part of what I changed but should be using that function to avoid issues

Reply:
added escape function as suggested.